doc/shelley-genesis.md

```diff
@@ -262,68 +379,54 @@ Available options:
   --verification-key-file FILEPATH
                            Input filepath of the verification key.
 ```
-So lets do that for the UTxO key
+So let's do that for the UTxO key
 ```
 $ cardano-cli shelley genesis initial-addr \
-  --verification-key-file example/utxo-keys/utxo1.vkey 
-820658207c3c942eaa39dd0aec74415312cf80dfc92b34fa1d0e1c2fd2499ee105219305
+    --verification-key-file example/utxo-keys/utxo1.vkey
+  820658207c3c942eaa39dd0aec74415312cf80dfc92b34fa1d0e1c2fd2499ee105219305
 ```
-We can copy that into the `InitialFunds`
+And if we compare this with the `initialFunds` from the generated file we see
 ```
-  "InitialFunds": {
+  "initialFunds": {
     "820658207c3c942eaa39dd0aec74415312cf80dfc92b34fa1d0e1c2fd2499ee105219305":
-      1000000,
+      0
   }
 ```
-This means we'll start with 1000000 lovelace in a special genesis UTxO at that
+This means we'll start with 0 lovelace in a special genesis UTxO at that
 address.
 
-We need to set the `MaxLovelaceSupply` to be at least as big as the sum total
-from our `InitialFunds`, but it can be bigger to allow for monetary expansion
-used later for stake rewards.
-
-Finally the `StartTime`, which is the agreed time of slot zero, needs to be
-set to a time in the near future or near past. It cannot be too far in the past
-otherwise the system would start having missed a very large number of slots.
-
-Here's our final example
+Ok, so zero lovelace is not that useful. We can however edit the
+`genesis.spec.json` and set the `maxLovelaceSupply` there, or we specify the
+initial supply when we re-generate the genesis file. Either way, it will be
+split equally between all the utxo keys.
 ```
-{
-   "StartTime":"2020-05-07T00:00:00Z",
-   "NetworkMagic":42,
-   "ProtocolMagicId":42,
-
-   "SlotLength":1,
-   "ActiveSlotsCoeff": 5.0e-2,
-   "DecentralisationParam":1,
-   "SecurityParam":2160,
-   "EpochLength":21600,
-   "SlotsPerKESPeriod":86400,
-   "MaxKESEvolutions":90,
-   "UpdateQuorum":5,
-
-   "MaxMajorPV":1000,
-   "MaxBodySize":16384,
-   "MaxHeaderSize":1400,
-
-    "GenDelegs": {
-      "a4d927a8e50c7a51e0f7d41a75057073cd2fc49bfc87a44891a8a9f80800cd8a":
-        "b2ee836b2b92fd3dd5e4228c943d8854e673c0510983956ce4f4ea7afcd9f761"
-    },
-
-    "InitialFunds": {
-      "820658207c3c942eaa39dd0aec74415312cf80dfc92b34fa1d0e1c2fd2499ee105219305":
-        1000000,
-    }
+$ cardano-cli shelley genesis create \
+    --genesis-dir example/
```

Comment:
missing `\` at the end: `--genesis-dir example/ \`

Reply:
thanks